modules/dxp/apps/saml/saml-opensaml-integration/src/test/java/com/liferay/saml/opensaml/integration/internal/BaseSamlTestCase.java

```diff
@@ -328,7 +327,8 @@ protected void prepareServiceProvider(String entityId) {
 	protected IdentifierGenerationStrategyFactory
 		identifierGenerationStrategyFactory;
 	protected List<String> identifiers = new ArrayList<>();
-	protected LocalEntityManager localEntityManager;
+	protected KeyStoreCredentialResolver.KeyStoreLocalEntityManager
+		localEntityManager;
```

Comment:
Hey @GabrielHenriquedev, discard this change. Using the `LocalEntityManager` is enough here since the `KeyStoreCredentialResolver.KeyStoreLocalEntityManager` **is an** `LocalEntityManager `. Thx